plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py

```diff
@@ -274,7 +274,9 @@ def augmentation(
 
         # some assertions
         assert peft_config is not None, "need peft_config to install PEFT adapters"
-
+        assert (
```

Comment:
can you put link to this issue in comments https://github.com/foundation-model-stack/fms-acceleration/issues/84